impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -35,6 +35,415 @@ define(["./_focus"], function(focusUtil) {
     delete: 46
   };
 
+  function makeAccessibleSelect(elem) {
+    var listboxNode = elem;
+    listboxNode.setAttribute('role', 'listbox');
+    listboxNode.setAttribute('tabindex', '0');
+    listboxNode.setAttribute('aria-activedescendant', '');
+    listboxNode.setAttribute('aria-multiselectable', 'true');
+    var activeDescendant = listboxNode.getAttribute(
+        'aria-activedescendant'
+    );
+    var multiselectable = listboxNode.hasAttribute('aria-multiselectable');
+    var startRangeIndex = 0;
+    var keysSoFar = '';
+    var handleFocusChange = function () {};
+    var handleItemChange = function () {};
+    registerEvents();
+
+    /**
+     * @description
+     *  If there is no activeDescendant, focus on the first option
+     */
+    function setupFocus() {
```

Comment:
This method's name `setupFocus` does not seem to match what it is doing.
You could call it `onFocus` and `onBlur`, or, to reveal what is being done, something like `setupActiveDescendantOnFocus` and `setupActiveDescendantOnBlur`.

In any case, I think a different solution would be best, perhaps. Always mark the active descendant with a specific class, e.g. `focused`. Then, in CSS, you could target that element, in the same with-focus-state, with a selector such as `:focus-visible .focused`. No additional event listening required: just update the active element's class at the same time and places when and where the `aria-activedescendant` is updated.